imageprocessor-api/src/test/java/pl/edu/uj/kimage/api/ApiTest.java

```diff
@@ -0,0 +1,39 @@
+package pl.edu.uj.kimage.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ApiTest {
```

Comment:
I thought about it. First thing we would need to establish some package name to scan for this classes. Second is that we would either need to write some funny reflection to do the scanning, or plug in some framework which would do this for us. Even more difficult is to fill this model with some data. Of course  it is all doable, but as I was looking on these solutions it looked like slight overkill for this purpose.